backend/src/main/java/com/crm/exception/user/NoSuchAccountException.java

```diff
@@ -0,0 +1,12 @@
+package com.crm.exception.user;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.NOT_FOUND)
```

Comment:
I think it should extend RuntimeException because CrmLoginException is handled with a code 400, but this exception has a different code. NoSuchAccountException should have a separate method for handling (with a response status code 404).

Reply:
Thanks for this advice. It's really working fine!